File: backend/src/main/java/com/pofo/backend/domain/resume/activity/award/dto/AwardRequest.java
package com.pofo.backend.domain.resume.activity.award.dto;

import jakarta.validation.constraints.NotBlank;
import jakarta.validation.constraints.NotNull;
import java.time.LocalDate;
import lombok.Getter;
import lombok.Setter;

@Getter
@Setter
public class AwardRequest {
    @NotBlank
    private String name;
    @NotBlank
    private String institution;
    @NotNull
    private LocalDate awardDate;

}

File: backend/src/main/java/com/pofo/backend/domain/resume/activity/award/dto/AwardRequest.kt
package com.pofo.backend.domain.resume.activity.award.dto

import jakarta.validation.constraints.NotBlank
import jakarta.validation.constraints.NotNull
import java.time.LocalDate


data class AwardRequest(
    @field:NotBlank val name: String,
    @field:NotBlank val institution: String,
    @field:NotNull val awardDate: LocalDate
)